.travis.yml

```diff
@@ -8,10 +8,6 @@ node_js:
 
 sudo: false
 
-cache:
```

Comment:
I've ran into many issues in the past with caching node_modules with Travis where it ends up getting cached in a bad state. When this happens it is not obvious that clearing the cache will resolve issues and many hours end up being wasted digging into the issue.